packages/client/components/Dashboard/LeftDashNavItem.tsx

```diff
@@ -44,48 +54,38 @@ const Label = styled('div')({
   wordBreak: 'break-word'
 })
 
+const iconLookup = {
+  arrowBack: <ArrowBack />,
+  creditScore: <CreditScore />,
+  forum: <Forum />,
+  history: <History />,
+  playlist_add_check: <PlaylistAddCheck />,
+  add: <Add />,
+  exit_to_app: <ExitToApp />,
+  group: <Group />,
+  warning: <Warning />
+}
+
 interface Props {
   className?: string
   onClick?: () => void
   label: string
   href: string
   //FIXME 6062: change to React.ComponentType
-  icon: string
-}
-
-const getIsActive = (href: string) => {
-  const {pathname} = window.location
-  const slashlessPath = pathname.endsWith('/') ? pathname.slice(0, -1) : pathname
-  if (href === '/me') return slashlessPath === href
-  if (href.startsWith('/newteam')) {
-    return href.startsWith(slashlessPath)
-  }
-  return slashlessPath.startsWith(href)
+  icon: keyof typeof iconLookup
 }
 
 const LeftDashNavItem = (props: Props) => {
   const {className, label, icon, href, onClick} = props
-  const {history} = useRouter()
-  const isActive = getIsActive(href)
+  const history = useHistory()
+  const match = useRouteMatch(href)
```

Comment:
Looks like we can use `useRouteMatch` here as well to simplify the `isActive` logic